ansible/pbTestScripts/testJDK.sh

```diff
@@ -11,13 +11,28 @@ else
 	ls -ld $HOME/openjdk-build/workspace/build/src/build/*/images/*
 fi
 
+# Special case for Solaris. See: https://github.com/adoptium/infrastructure/pull/2405#issuecomment-999498345
+if [[ $(uname) == "SunOS" ]]; then
+	export PATH="/opt/csw/bin:/usr/local/bin:${PATH}"
+fi
+
+
 mkdir -p $HOME/testLocation
 [ ! -d $HOME/testLocation/aqa-tests ] && git clone https://github.com/adoptium/aqa-tests.git $HOME/testLocation/aqa-tests
 # cd to aqa-tests as required by https://github.com/adoptium/aqa-tests/issues/2691#issue-932959102
 cd $HOME/testLocation/aqa-tests
 $HOME/testLocation/aqa-tests/get.sh
 cd $HOME/testLocation/aqa-tests/TKG || exit 1
-export BUILD_LIST=functional
-$MAKE_COMMAND compile
-# Runs this test to check for prerequisite perl modules
-$MAKE_COMMAND _MBCS_Tests_pref_ja_JP_linux_0
+
+# Solaris runs a different test to Linux.
+# See: https://adoptium.slack.com/archives/C53GHCXL4/p1641311568115100?thread_ts=1641296204.114900&cid=C53GHCXL4
+if [[ $(uname) == "SunOS" ]]; then
+	export BUILD_LIST=openjdk
+	$MAKE_COMMAND compile
+	$MAKE_COMMAND _jdk_math
```

Comment:
May be worth just going back to `_MachineInfo` if that test is Linux-specific, as we're just looking to do a quick sniff test to ensure the thing has built properly.

Reply:
Fixed! :-) It'll be much faster now, hopefully!